explorer/scan.go

```diff
@@ -23,7 +23,7 @@ func isSynced(b Block) bool {
 }
 
 func (e *Explorer) waitForSync() error {
-	ticker := time.NewTicker(10 * time.Second)
+	ticker := time.NewTicker(1 * time.Second)
```

Comment:
This is fine if we really want to check every second. We don't generally tweak production parameters to make tests run faster.

Reply:
Reindexing from scratch takes 1-2 days, so it is not necessary to check this frequently.  But even with SQLite being single threaded I doubt it costs very much to lower it to 1s.